packages/components-css/link-css/src/_mixin.scss

```diff
@@ -56,13 +54,10 @@
    * however this doesn't guarantee significant contrast between `outline-color` and `inverse-outline-color`.
    * That's why we use `Highlight` vs `HighlightText`.
```

Comment:
I think this comment is now also not in the right place and should be moved to the generic focus styles? (Maybe we can copy it into the generic focus state issue?)